sdcm/cql_stress_cassandra_stress_thread.py

```diff
@@ -0,0 +1,158 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+
+import logging
+import re
+import os
+import contextlib
+from sdcm.loader import CqlStressCassandraStressExporter
+from sdcm.prometheus import nemesis_metrics_obj
+from sdcm.sct_events.loaders import CqlStressCassandraStressEvent
+from sdcm.stress_thread import CassandraStressThread
+from sdcm.utils.common import SoftTimeoutContext
+from sdcm.utils.docker_remote import RemoteDocker
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class CqlStressCassandraStressThread(CassandraStressThread):
+    DOCKER_IMAGE_PARAM_NAME = 'stress_image.cql-stress-cassandra-stress'
+
+    def __init__(self, loader_set, stress_cmd, timeout, stress_num=1, keyspace_num=1, keyspace_name='', compaction_strategy='',  # pylint: disable=too-many-arguments
+                 profile=None, node_list=None, round_robin=False, client_encrypt=False, stop_test_on_failure=True,
+                 params=None):
+        super().__init__(loader_set=loader_set, stress_cmd=stress_cmd, timeout=timeout,
+                         stress_num=stress_num, node_list=node_list,  # pylint: disable=too-many-arguments
+                         round_robin=round_robin, stop_test_on_failure=stop_test_on_failure, params=params,
+                         keyspace_num=keyspace_num, keyspace_name=keyspace_name, profile=profile,
+                         client_encrypt=client_encrypt, compaction_strategy=compaction_strategy)
+
+    def create_stress_cmd(self, cmd_runner, keyspace_idx, loader):  # pylint: disable=too-many-branches
+        stress_cmd = self.stress_cmd
+
+        if "no-warmup" not in stress_cmd:
+            # add no-warmup to stress_cmd if it's not there. See issue #5767
```

Comment:
Which project's issue?

Reply:
this comment seem to be copied from somewhere else...
I don't know what the issue, but we shouldn't be using warmup, it's doesn't have any meaning for scylla, its something todo with cassandra java implementation ontop of the JVM, that need time to "stabilize"